src/panels/lovelace/cards/hui-entities-card.ts

```diff
@@ -296,12 +296,11 @@ class HuiEntitiesCard extends LitElement implements LovelaceCard {
 
   private renderEntity(entityConf: LovelaceRowConfig): TemplateResult {
```

Comment:
@bramkragten shouldn't the type be `EntitiesCardEntityConfig` here?  The Lovelace vs card types have always baffled me.

Reply:
Looks like that's what it used to be before #7134... 😕

Reply:
Can't entityConf here be one of the special row types, like a `type: divider` ? It's not necessarily an entity.

Reply:
> Can't entityConf here be one of the special row types, like a `type: divider` ? It's not necessarily an entity.

Yes, but my point is that the `EntityConfig` type is wrong when it's an entity - `EntitiesCardEntityConfig` is what is schemed in the docs and UI.

Reply:
You mean that `LovelaceRowConfig` should have `EntitiesCardEntityConfig` in it instead of `EntityConfig`? Then yes.

Reply:
> You mean that `LovelaceRowConfig` should have `EntitiesCardEntityConfig` in it instead of `EntityConfig`? Then yes.

Yes, and making that change doesn't cause errors, but I see a bunch of type assertions and inconsistencies that probably ought to be checked as part of that change.  Keeping the scope narrow here, I kept it as is and just adjusted to make sure `false` is passed.